phys/module_fr_fire_atm.F

```diff
@@ -124,6 +168,8 @@ SUBROUTINE fire_tendency( &
    REAL :: fact_g, fact_c
    REAL :: alfg_i, alfc_i
 
+   REAL, DIMENSION( ims:ime,kms:kme,jms:jme ) :: prop_heat !proportion of heat to be released fro TG dist.
```

Comment:
If prop_heat is a local array, it is more standard in WRF to define it using tile indeces

Reply:
Changed.